packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -827,6 +830,38 @@ describe('plugin-meetings', () => {
               },
             });
           });
+
+          //TODO: can't seem to make this pass
```

Comment:
This test is failing because the assert is checked before the finally() code is executed. It's because `handleLocusEvent()` doesn't return the promise from the call to `this.create()`

Reply:
how can I test this then?

Reply:
you could change `handleLocusEvent()` to return a promise - the production code ignores its return value anyway, but just adding `await testUtils.flushPromises();` in the test before the assert also fixes the test and doesn't affect the production code, so probably better to just do that